js/packages/common/src/contexts/meta/meta.tsx

```diff
@@ -81,6 +84,14 @@ export function MetaProvider({ children = null as any }) {
     }
   }
 
+  const patchState: MetaContextState['patchState'] = temp => {
+    const newState = merge({}, state, temp);
+    newState.store = temp.store ?? state.store;
```

Comment:
The store is the only non-collection value tracked by meta state so needs to default to the previous state if not in the patch state snippet.